components/Table/Table.js

```diff
@@ -95,12 +118,186 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  renderDropDownMenu(menuBtn, col, cell) {
+    const dropDownMenu = document.createElement('div');
+    dropDownMenu.classList.add('drop-down__menu');
+    this.menuDropDownItems = this.menuConfig.map((menuElement) => {
+      if (!col.sortable) {
```

Comment:
From the `col` argument here you need to know is it sortable or not only, let's create just a boolean flag